decrypt/decrypt.go

```diff
@@ -79,26 +79,41 @@ func Decrypt(args []string) error {
 		return err
 	}
 
+	var warnings []string
 	// Check that all the encrypted files exist, and all the unencrypted don't
 	for _, file := range files {
 		// check that the input file exists and is readable
 		if !helpers.FileIsReadable(file.Encrypted) {
-			return fmt.Errorf("cannot read input file %s", file.Encrypted)
+			warnings = append(warnings, fmt.Sprintf("Warning: cannot read input file %s", file.Encrypted))
```

Comment:
This is a bit of a semantic problem, should be just continue print warnings if a file is not readable? Either that the file doesn't exists or that there are permission problems.

Reply:
Checking whether a file is readable without determining the cause, whether due to its absence or permission issues, is used elsewhere in the code. If this is needed, I think it would be better to address this in a new refactoring issue.

Reply:
That might be but with this PR the semantics are changed, abort -> warning so the actual root cause should be shown to the user (file missing or file not readable).

We should add an issue about this since the changes will affect other parts of the code.